promises.cf

```diff
@@ -57,6 +57,10 @@ body common control
 
       version => "CFEngine Promises.cf 3.7.0";
 
+      cfengine_3_7.(debian||redhat)::
+          package_inventory => { $(package_module_knowledge.platform_default) };
+          package_module => $(package_module_knowledge.platform_default);
```

Comment:
Agree, that's very nasty, and it is the first file everyone who tries CFEngine will see, I really don't think we should split it. The problem is that even if the section is commented, you cannot enable it as long as you have any 3.6 hosts. In the transition phase I assume this will be quite common.